packages/ai-core/src/browser/frontend-prompt-customization-service.ts

```diff
@@ -154,23 +460,29 @@ export class FrontendPromptCustomizationServiceImpl implements PromptCustomizati
     }
 
     protected removePromptTemplateSuffix(filename: string): string {
-        const suffix = PROMPT_TEMPLATE_EXTENSION;
-        if (filename.endsWith(suffix)) {
-            return filename.slice(0, -suffix.length);
+        for (const ext of this.templateExtensions) {
+            if (filename.endsWith(ext)) {
+                return filename.slice(0, -ext.length);
+            }
+        }
+        // If no matching extension found, try the default one
+        if (filename.endsWith(PROMPT_TEMPLATE_EXTENSION)) {
+            return filename.slice(0, -PROMPT_TEMPLATE_EXTENSION.length);
```

Comment:
It's inconsistent to hard code this value here, but not for all other cases. We should either hard code the `.prompttemplate` extension for all use cases, or remove this code here.

Reply:
I made sure now, that the default .prompttemplate is always in this list.